docs/TESTS.md

```diff
@@ -16,11 +16,11 @@ Words written in this way will implicitly be easily `unit-test`able.
 
 Unit tests (usually a bunch of assertions like above) go in a file called `vocab-name-tests.factor` beside your implementation `vocab-name.factor`. This file is already created for you by Exercism, but would normally need to be created by hand, or by `"exercise" scaffold-tests`.
 
-Get the [`tools.test` wrapper library for Exercism](https://github.com/catb0t/exercism.testing), and put its `exercism` subfolder inside Factor's `work` folder, such that `testing.factor` is located at `resource:work/exercism/testing/testing.factor`.
+Get the [`tools.test` wrapper library for Exercism](https://github.com/catb0t/exercism.factor), and put its `exercism` subfolder inside Factor's `work` folder, such that `testing.factor` is located at `resource:work/exercism/testing/testing.factor`.
 
 When the current directory is your `exercism/factor` exercises folder:
 
 * Run a vocabulary's tests from the listener with `USE: exercism.testing "exercise-name" run-exercism-test`, or from the command-line with `factor -run=exercism.testing exercise-name`.
 * Run all tests for all exercises with `USE: exercism.testing run-all-exercism-tests` or from the command-line with `factor -run=exercism.testing run-all`.
 
-For more information, see the Factor documentation on [Unit testing](http://docs.factorcode.org/content/article-tools.test.html), and [`exercism.testing`](https://github.com/catb0t/exercism.testing)'s documentation with `"exercism.testing" help`.
+For more information, see the Factor documentation on [Unit testing](http://docs.factorcode.org/content/article-tools.test.html), and [`exercism.testing`](https://github.com/catb0t/exercism.factor/tree/HEAD/exercism/testing)'s documentation with `"exercism.testing" help`.
```

Comment:
This arguably doesn't need a change? Open to suggestion

Reply:
Intead, I think this should be merged in or set up to be exercism/factor not catb0t's repository.  It likely was meant as a proposal to come in here eventually.  But it is 4 years old, I think.

So set this up to be less reliant on the external repository.

Reply:
I might be wrong here though, as that repository looks like it is intended as an "add-on" to exercism.

Reply:
It does look old, for sure, but I think it still works. I'm thinking fixing links that currently 404 is a good "for now" step.

There are also links to this same repo in other md files. And really there are more things that I think should be changed (mentions of 0.98 not available yet, installation and setup instructions, etc) and I'm not sold that we should be allowing word names defined in other vocabularies anyway, since the whole goal is to be idiomatic in the language.

What do you think about me opening a separate issue to revise the dev/test setup in general?

Reply:
https://github.com/exercism/factor/issues/97